src/commands/mod.rs

```diff
@@ -900,6 +900,9 @@ struct SparseSetArgs {
     /// Include no files in the working copy (combine with --add)
     #[arg(long)]
     clear: bool,
+    /// Edit patterns with $EDITOR
```

Comment:
I chose for `--edit` to be compatible with all other flags, that way you can do `jj sparse set --clear --edit` and you get a file with no patterns. Instead we could make it conflict with other flags to avoid confusion, but IMO this is fine this way.

Reply:
Yes, seems fine to me.